packages/workflow-core/src/lib/workflow-runner.ts

```
@@ -769,13 +760,16 @@
     }
   }
 
-  private async __invokeApiPlugin(apiPlugin: HttpPlugin) {
+  private async __invokeApiPlugin(apiPlugin: HttpPlugin, additionalContext?: AnyRecord) {
     // @ts-expect-error - multiple types of plugins return different responses
-    const { callbackAction, responseBody, error } = await apiPlugin.invoke?.({
-      ...this.context,
-      workflowRuntimeConfig: this.#__config,
-      workflowRuntimeId: this.#__runtimeId,
-    });
+    const { callbackAction, responseBody, error } = await apiPlugin.invoke?.(
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)